chrome/content/zotero/xpcom/translation/translators.js

```diff
@@ -368,39 +368,47 @@ Zotero.Translators = new function() {
 	
 	/**
 	 * @param	{String}		metadata
-	 * @param	{String}		metadata.translatorID		Translator GUID
-	 * @param	{Integer}		metadata.translatorType		See TRANSLATOR_TYPES in translate.js
-	 * @param	{String}		metadata.label				Translator title
-	 * @param	{String}		metadata.creator			Translator author
-	 * @param	{String|Null}	metadata.target				Target regexp
-	 * @param	{String|Null}	metadata.minVersion
-	 * @param	{String}		metadata.maxVersion
-	 * @param	{String|undefined}	metadata.configOptions
-	 * @param	{String|undefined}	metadata.displayOptions
-	 * @param	{Integer}		metadata.priority
-	 * @param	{String}		metadata.browserSupport
-	 * @param	{Boolean}		metadata.inRepository
-	 * @param	{String}		metadata.lastUpdated		SQL date
+	 *   @param	{String}		metadata.translatorID		Translator GUID
+	 *   @param	{Integer}		metadata.translatorType		See TRANSLATOR_TYPES in translate.js
+	 *   @param	{String}		metadata.label				Translator title
+	 *   @param	{String}		metadata.creator			Translator author
+	 *   @param	{String|Null}	metadata.target				Target regexp
+	 *   @param	{String|Null}	metadata.minVersion
+	 *   @param	{String}		metadata.maxVersion
+	 *   @param	{String|undefined}	metadata.configOptions
+	 *   @param	{String|undefined}	metadata.displayOptions
+	 *   @param	{Integer}		metadata.priority
+	 *   @param	{String}		metadata.browserSupport
+	 *   @param	{Boolean}		metadata.inRepository
+	 *   @param	{String}		metadata.lastUpdated		SQL date
```

Comment:
Idk how you will feel about this, but I think it helps a little bit with such long lists. Also, the spacing looks nice, but maybe we should use spaces instead of tabs, since it doesn't really look good with other tab width settings. (I use 2 spaces/tab, for instance)